ports/samd/modules/_boot.py

```diff
@@ -2,18 +2,17 @@
 import uos
 import samd
 
-samd.Flash.flash_init()
 bdev = samd.Flash()
 
 # Try to mount the filesystem, and format the flash if it doesn't exist.
 fs_type = uos.VfsLfs2 if hasattr(uos, "VfsLfs2") else uos.VfsLfs1
 
 try:
-    vfs = fs_type(bdev)
+    vfs = fs_type(bdev, progsize=256)
 except:
-    fs_type.mkfs(bdev)
-    vfs = fs_type(bdev)
+    fs_type.mkfs(bdev, progsize=256)
+    vfs = fs_type(bdev, progsize=256)
 uos.mount(vfs, "/")
 
+del vfs, fs_type, bdev, uos, samd, progsize
```

Comment:
I'm surprised this doesn't crash... `progsize` isn't defined!

Also, why not delete after the `gc.collect()`, so that `gc` can also be deleted?